cinnamon-remix/cinnamon-remix-calamares.desktop.in

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
+_Name=Install cinnamon-remix 19.10
```

Comment:
Is your human-readable name for the flavor really `cinnamon-remix` or is it `Cinnamon Remix`?

Reply:
cinnamon-remix